rfcs/components/moving-from-focus-zone-to-tabster.md

```diff
@@ -0,0 +1,130 @@
+  - _Equivalent in `Tabster`:_ Use `findLastFocusable` from `useFocusFinders` in `@fluentui/react-tabster` instead.
+
+#### IFocusZoneProps
+
+- `isCircularNavigation?: boolean`
+  - _Description:_ If set, the `FocusZone` will cycle to the beginning of the targets once the user navigates to the next target while at the end, and to the end when navigating to the previous target while at the beginning.
+  - _Equivalent in `Tabster`:_ `useArrowNavigationGroup` from `@fluentui/react-tabster` can be used to allow for cyclic navigation, specified by passing the `circular` prop as part of the hook's `options` argument.
+
+### What is covered in `FocusZone` that is partially covered by `Tabster`
+
+#### IFocusZoneProps
+
+- `direction?: FocusZoneDirection`
+  - _Description:_ Defines which arrows to react to.
+  - _Notes:_ `FocusZoneDirection` is an `enum` with the following values:
+    - `vertical`: Reacts to Up/Down arrows.
+    - `horizontal`: Reacts to Left/Right arrows.
+    - `bidirectional`: Reacts to all arrows.
+    - `domOrder`: Reacts to all arrows by navigating to the next item in DOM by pressing Right/Down arrow keys and to the previous item in the DOM by pressing Left/Up arrow keys. Left and Right arrow keys are swapped in RTL mode.
+  - _Partial equivalent in `Tabster`:_ `useArrowNavigationGroup` from `@fluentui/react-tabster` can be used to allow for arrow navigation for either a vertical or horizontal axis, specified by passing the `axis` prop as part of the hook's `options` argument.
+  - _What is missing in `Tabster`:_ `bidirectional` and `domOrder` navigation are not currently available via `useArrowNavigationGroup`.
+- `handleTabKey?: FocusZoneTabbableElements`
+  - _Description:_ Allows tab key to be handled to tab through a list of items in the `FocusZone`. An unfortunate side effect is that users will not be able to tab out of the `FocusZone` and have to hit some other key.
+  - _Notes:_ `FocusZoneTabbableElements` is an `enum` with the following values:
+    - `none`: Tabbing is not allowed.
+    - `all`: All tabbing is allowed.
+    - `inputOnly`: Tabbing is allowed only on input elements.
+  - _Partial equivalent in `Tabster`:_ `Tabster` allows the user to specify the navigation type of the `Mover` to be handled via arrow keys, the tab key, or both.
+  - _What is missing in `Tabster`:_ And abstraction similar to `useArrowNavigationGroup` would be nice to have. Even then, there is no equivalent currently for `inputOnly` tabbing.
+
+### What is covered in `FocusZone` that is not covered by `Tabster`
+
+#### Props that need no equivalent
+
+##### IFocusZoneProps
+
+- `componentRef?: IRefObject<IFocusZone>`
+  - _Description:_ Optional callback to access the `IFocusZone` interface used instead of `ref` for accessing the public methods and properties of the component.
+  - _Why is no equivalent needed?_ `Tabster` is not a component but a set of utilities, so it has no need for a `ref`.
```

Comment:
This does not necessarily rule creating a `FocusZone`, especially for partner migration purposes. IMO it would be great if we could just end up providing helpful utilities to partners instead of an all powerful component

Reply:
Agreed, I think a `FocusZone` that leverages `Tabster` might be necessary if we can't find a solution to partners that use class-based components migrating to utilities.